jq-static/tests/test.bats

```diff
@@ -0,0 +1,6 @@
+TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
```

Comment:
[nit] What about this? 😉  
```suggestion
TEST_PKG_VERSION="$(cut -d/ -f3 <<<"${TEST_PKG_IDENT}")"
```

Reply:
I don't think this is strictly necessary or required and I'm not sure I understand the tradeoffs, so I don't want to use this (right now). I've seen this in a couple of other suggestions though. Perhaps we need to start a design doc on best practices in tests?

Reply:
Totally neither necessary nor required 😄